### ISSUE TITLE
Setting width

### ISSUE THREAD

**Issue by terreb** [manijak/nativescript-carousel#42]
This is not an issue, but proposal!

Another great addition to this awesome plugin would be the ability to set the width of the carousel page and distance between pages.

**Comment by manijak** [comment]
Thanks for the feedback. 
Any particular reason why you would want a feature like this? Is the idea to see the "ends" of previous and next slide?

**Comment by terreb** [comment]
Yes exactly, I'm particularly interested in having the parts of side slides seen as per design:) But other people might be interested for example in having multiple slides visible at the same time on the screen. i.e. if there would be an ability to set slide width at 30% then 3 slides can be seen on the page.

**Comment by asaph26** [comment]
Is there a possibility to have this added?

**Comment by kambojaz** [comment]
Are there any solutions?

**Comment by dotnetdreamer** [comment]
i also need this feature. It would be great to have this...

**Comment by Vinaykummar** [comment]
please add the spacing property between the slides so that user knows that there is more content to view

**Comment by manijak** [comment]
Currently this is not a priority, but PRs are welcome :)

**Comment by Vinaykummar** [comment]
This is exactly what i was asking
https://camo.githubusercontent.com/7590ec7e6b705a6e8e381397247d576c6db72147/68747470733a2f2f692e696d6775722e636f6d2f653157625a63752e676966

**Comment by Vinaykummar** [comment]
You can take this as a reference from the reactnative plugin.
https://github.com/archriss/react-native-snap-carousel

**Comment by HaikelPixpay** [comment]
> Thanks for the feedback.
> Any particular reason why you would want a feature like this? Is the idea to see the "ends" of previous and next slide?

I would like if possible

**Comment by tigerdi** [comment]
Absolutely agree that there needs to be a nativescript carousel plugin that enables you to see the beginning of the next slide and the end of the previous slide - exactly like the react native plugin - or similar to the iCarousel pod. This plugin looks to be the best carousel on offer for Nativescript at the moment, so would be great if this feature was added.

At the very least, would you be able to point me in the direction of how to do this myself?

**Comment by Lehren** [comment]
Any news? Would love this

**Comment by coltkenn2658** [comment]
did this feature ever get implemented?? @manijak
